_data/tools.yml

```diff
@@ -1042,10 +1045,10 @@
     - data-validators
     - description-validators
   github: https://github.com/isa-group/oas-tools
-  language: Node.js
+  language: Node.js + Express
```

Comment:
Let's not put frameworks in here. NodeJS is kinda unfortunately a language as its a whole environment beyond just JS/TS but I dont want to list Express and all the other WAFs things work with or it'll get bloated.

Reply:
Framework really ought to be **somewhere**. If you're using NestJS or Meteor, a tool for Express won't do you any good.